src/main/java/com/doyak/reflector/dto/response/PostResponse.java

```diff
@@ -2,4 +2,37 @@
 
 import com.doyak.reflector.domain.enums.Site;
 
-public record PostResponse(Long postId, Site site, Integer level, String title, String content) {}
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+public class PostResponse {
+	
+	@Builder
+    @NoArgsConstructor(access = AccessLevel.PROTECTED)
+    @AllArgsConstructor
+    @Getter
+	public static class PostInfo {
+		private Long postId;
+		private Site site;
+		private Integer level;
+		private String title;
+		private String content;	
+		private String author;
+		private String createdAt;
+	    private String updatedAt;
+	}
+	
+	@Builder
+    @NoArgsConstructor(access = AccessLevel.PROTECTED)
+    @AllArgsConstructor
+    @Getter
+	public static class PosOverview {
```

Comment:
```suggestion
	public static class PostOverview {
```
오타 발견했습니다! 오타가 아니라 의도하신거라면 ... 죄송 ...😶‍🌫️

Reply:
> 오타 발견했습니다! 오타가 아니라 의도하신거라면 ... 죄송 ...😶‍🌫️

오타입니다😵‍💫 수정해서 올렸습니다!!